tests/tt_eager/python_api_testing/unit_testing/test_optimized_conv_v2.py

```diff
@@ -201,3 +200,124 @@ def test_optimized_conv_v2(
     print("Passing=", passing_pcc)
     print("Info=", info)
     assert passing_pcc
+
+
+def test_simple(
+    use_program_cache,
+    device,
+):
```

Comment:
I can remove we you want.  I found this test useful for just one-off debugging, but I can keep it local.

Reply:
I ended up removing it since I realized we wanted to deprecate this test